res/entities/Player.json

```diff
@@ -4,13 +4,13 @@
     {
       "componentType": "Player"
     },
-	{
+	  { 
       "componentType": "Sprite",
       "src": "entity/player_modelDefault",
-	  "origin": {
-		"x": 18,
-		"y": 48
-	  },
+	    "origin": {
+		    "x": 15,
```

Comment:
If we do that for all the files and don't check if it's correct, there's a big chance it will mess up in a lot of places. These tools are usually not perfect.